tests/bgp/test_bgp_bbr.py

```diff
@@ -50,8 +50,7 @@ def prepare_bbr_config_files(duthosts, rand_one_dut_hostname):
 
     yield
 
-    duthost.copy(src="./bgp/templates/del_bgp_bbr_config.json", dest='/tmp/del_bgp_bbr_config.json')
-    duthost.shell("configlet -d -j {}".format("/tmp/del_bgp_bbr_config.json"))
+    delete_running_config("./bgp/templates/del_bgp_bbr_config.json", duthost, is_json=False)
 
```

Comment:
Does this function did the same thing as original code, but we use a single function call for better maintains?

Reply:
Sure, this function is added in PR https://github.com/sonic-net/sonic-mgmt/pull/8901

Reply:
I see, thanks for the information.